src/test/java/games/strategy/triplea/delegate/MoveDelegateTest.java

```diff
@@ -50,7 +51,9 @@ public void testNotUnique() {
     route.setStart(egypt);
     route.add(eastAfrica);
     final List<Unit> units = armour.create(1, british);
-    units.addAll(units);
+    // Wrapping this line into an unmodifiable List
+    // is neccessary to avoid the errorprone error
+    units.addAll(Collections.unmodifiableList(units));
```

Comment:
I think it would be clearer to just find another way to add duplicate units to the collection.  Using the `UnitType#create(PlayerID)` overload seems a better choice here:

```diff
diff --git a/src/test/java/games/strategy/triplea/delegate/MoveDelegateTest.java b/src/test/java/games/strategy/triplea/delegate/MoveDelegateTest.java
index 00f68f8ed..d20ae3f4d 100644
--- a/src/test/java/games/strategy/triplea/delegate/MoveDelegateTest.java
+++ b/src/test/java/games/strategy/triplea/delegate/MoveDelegateTest.java
@@ -7,8 +7,8 @@ import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
-import java.util.Collections;
 import java.util.List;
 
 import org.junit.jupiter.api.BeforeEach;
@@ -50,10 +50,8 @@ public class MoveDelegateTest extends DelegateTest {
     final Route route = new Route();
     route.setStart(egypt);
     route.add(eastAfrica);
-    final List<Unit> units = armour.create(1, british);
-    // Wrapping this line into an unmodifiable List
-    // is neccessary to avoid the errorprone error
-    units.addAll(Collections.unmodifiableList(units));
+    final Unit unit = armour.create(british);
+    final List<Unit> units = Arrays.asList(unit, unit);
     final String results = delegate.move(units, route);
     assertError(results);
   }
```